src/modules/jcms_rest/src/JcmsRestServiceProvider.php

```diff
@@ -0,0 +1,22 @@
+<?php
+
+namespace Drupal\jcms_rest;
+
+use Drupal\Core\DependencyInjection\ContainerBuilder;
+use Drupal\Core\DependencyInjection\ServiceModifierInterface;
+
+/**
+ * Adds application/vnd.elife.annual-report-list+json as a known format.
+ */
+class JcmsRestServiceProvider implements ServiceModifierInterface {
+
+  /**
+   * {@inheritdoc}
+   */
+  public function alter(ContainerBuilder $container) {
+    if ($container->has('http_middleware.negotiation') && is_a($container->getDefinition('http_middleware.negotiation')->getClass(), '\Drupal\Core\StackMiddleware\NegotiationMiddleware', TRUE)) {
+      $container->getDefinition('http_middleware.negotiation')->addMethodCall('registerFormat', ['jcms_json', ['application/vnd.elife.annual-report-list+json']]);
```

Comment:
The content type is indeed `application/vnd.elife.annual-report-list+json` and anything after that is a parameter. That means we can accept various different versions and aren't tied to a single one.